v1-2/learn/how-to-deploy-and-run-ballerina-programs.md

```diff
@@ -315,190 +317,212 @@ For more information, see the [Docker build extension GitHub repo](https://githu
-db-url = "jdbc:mysql://mysql-server:3306/EMPLOYEE_RECORDS"
-db-username = "root"
-db-password = "root"
-key-store-password = "abc123"
-trust-store-password = "xyz123"
+keystore-password="ballerina"
+
+[john]
+name="John Doe"
+email="john@ballerina.com"
+
+[jane]
+name="Jane Doe"
+email="jane@ballerina.com"
 ```
 
-Here we have used `@kubernetes:Deployment` to specify the Docker image name that will be created as part of building this service. The `CopyFiles` field is used to copy the MySQL JAR file into the Ballerina `bre/lib` folder.
+Here we have used `@kubernetes:Deployment` to specify the Docker image name that will be created as part of building this service.
 
 The `@kubernetes:Service {}` annotation will create a Kubernetes service that will expose the Ballerina service running on a Pod.
 
 In addition, you can use `@kubernetes:Ingress`, which is the external interface to access your service (with path / and host name `ballerina.guides.io`).
 
 Minikube users please see the [Kubernetes Extension samples](https://github.com/ballerinax/kubernetes/tree/master/samples) for additional configurations required for Minikube.
 
+Create a folder called `security` and copy the [ballerinaKeystore.p12](https://github.com/ballerina-platform/ballerina-lang/raw/v1.2.0/examples/resources/ballerinaKeystore.p12). This is to secure the endpoint of the service.
+
 Now you can use the following command to build the Ballerina service that we developed above. This will also create the corresponding Docker image and the Kubernetes artifacts using the Kubernetes annotations that you have configured above.
 
 ```bash
-$ ballerina build data_backed_service.bal
+$ ballerina build user_retrieval_service.bal
 Compiling source
- 	data_backed_service.bal
+        user_retrieval_service.bal
 
 Generating executables
- 	data_backed_service.jar
+        user_retrieval_service.jar
 
 Generating artifacts...
 
-@kubernetes:Service                     - complete 1/1
-@kubernetes:Ingress                     - complete 1/1
-@kubernetes:Secret                      - complete 1/1
-@kubernetes:ConfigMap                	- complete 1/1
-@kubernetes:Docker                      - complete 3/3 
-@kubernetes:Deployment                  - complete 1/1
+        @kubernetes:Service                      - complete 1/1
+        @kubernetes:Ingress                      - complete 1/1
+        @kubernetes:Secret                       - complete 1/1
+        @kubernetes:ConfigMap                    - complete 1/1
+        @kubernetes:Deployment                   - complete 1/1
+        @kubernetes:Docker                       - complete 2/2 
+        @kubernetes:Helm                         - complete 1/1
+
+        Run the following command to deploy the Kubernetes artifacts: 
+        kubectl apply -f ./kubernetes
 
-Run the following command to deploy the Kubernetes artifacts: 
-kubectl apply -f ./kubernetes/
+        Run the following command to install the application using Helm: 

```

Comment:
```suggestion
        Execute the below command to install the application using Helm: 

```

Reply:
Since this is coming from the command, we will make this change in next release.